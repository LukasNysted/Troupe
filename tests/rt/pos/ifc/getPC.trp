let val x = if true raisedTo `{secret}` then _pc () else 1
 in printWithLabels x 
 end 