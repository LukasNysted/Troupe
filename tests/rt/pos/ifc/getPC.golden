2021-08-09T20:19:14.846Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
{secret}@{secret}%{}
>>> Main thread finished with value: ()@{secret}%{secret}
